charts/flyte-binary/README.md

```diff
@@ -109,10 +109,12 @@ Chart for basic single Flyte executable deployment
 | rbac.create | bool | `true` |  |
 | rbac.extraRules | list | `[]` |  |
 | rbac.labels | object | `{}` |  |
-| service.annotations | object | `{}` |  |
```

Comment:
While this is removed from the `values.yaml` file, we still keep support for it in the templates for backward compatibility.